Design rules/Policy Design Rules.md

```diff
@@ -0,0 +1,42 @@
+# Policy Design Rules
+
+In dit document beschrijven we hoe we in het Kenniscentrum Standaarden van VNG Realisatie omgaan met Design Rules. 
+Daarbij doelen we op de eigen Design Rules, de Design Rules die in de Nederlandse API Strategie zijn beschreven maar ook op de Extensions die op beide niveaus beschikbaar zijn.
+Extensions hebben geen officiële status, ze kunnen worden gezien als potentiële Design Rules.
+
+## Toepassing Design Rules
+
+Het toepassen van Design Rules en Extensions is aan de volgende regels gebonden:
+
+1. Bij het vervaardigen van API specificaties houden we ons aan [de Design Rules zoals gedefinieerd in de Nederlandse API Strategie](https://docs.geostandaarden.nl/api/API-Designrules/);
+2. Daarbij kijken we tevens in hoeverre [de Extensions van de Nederlandse API Strategie](https://docs.geostandaarden.nl/api/API-Strategie-ext/) van toepassing zijn. In een Design Decision van het project waarin de API specificatie wordt vervaardigd beschrijven we aan welke Extensions we ons houden;
+3. Tevens houden we ons aan [de Design Rules van VNG Realisatie](https://github.com/VNG-Realisatie/API-Kennisbank/blob/master/Design%20rules/readme.md);
+4. Tenslotte kijken we ook in hoeverre de Extensions op de Design Rules van VNG Realisatie van toepassing zijn voor het project. Ook hier bschrijven we in een Design Decision van het project weer aan welke Extensions we ons houden;
+
+## Het toevoegen, wijzigen of verwijderen van VNG Realisatie Design Rules
+
+Zoals al gesteld in [de Design Rules van VNG Realisatie](https://github.com/VNG-Realisatie/API-Kennisbank/blob/master/Design%20rules/readme.md) kunnen Design Rules onderhevig zijn aan voortschrijdend inzicht en technologische ontwikkelingen. 
+Het kan dus noodzakelijk blijken een Design Rule te wijzigen of verwijderen. Ook kan er behoefte aan een nieuwe Design Rule ontstaan. Over het algemeen zal deze behoefte in een project naar voren komen. 
+In dat geval handelen we als volgt:
+
+1. Er wordt in de onderliggende GitHub repository een issue ingediend met het label 'Design Rule'. 
+2. Indien het om een voorstel voor een nieuwe Design Rule gaat wordt deze zo duidelijk mogelijk beschreven, zo mogelijk voorzien van voorbeelden en ratio;
+3. Indien het om een voorstel tot wijziging gaat wordt beschreven welke Design Rule moet worden gewijzigd, wat die wijziging inhoudt en wat daarvoor de ratio is;
+4. Betreft het een verwijderingsvoorstel dan wordt beschreven welke Design Rule het betreft en wat de reden is van het verzoek;
+5. Het issue wordt assigned aan alle medewerkers van het Kenniscentrum Standaarden;
+6. Het issue wordt geagendeerd als onderwerp van het teamoverleg Kenniscentrum Standaarden waar wordt besloten of het voorstel wordt gehonoreerd. 
+In de periode tussen het opvoeren van het issue en het besluit wordt in de projecten niet vooruitgelopen op dat besluit. 
```

Comment:
Daarbij wordt van de collega's wel een pro-actieve houding verwacht in de discussie. Dit kan binnen een project veel haast hebben, dus het niet vooruitlopen op een beslissing kan problematisch zijn. Als we het niet eens zijn over de wijziging van de Design Rule of het opnemen van een Design Rule dan kan je wel vooruitlopen in een project en documenteer je dit fiet als een Design Decision van je project.

Reply:
Door je reactie realiseer ik me dat je wel vooruit mag lopen op een nieuwe Design Rule. Dat wordt immers een Design Decision van het project en als het goed is is deze nooit tegenstrijdig met bestaande Design Rules.

Dus alleen op een wijziging van een Design Rule mag je niet vooruitlopen mits aan de hierboven door jou geschetste voorwaarden wordt voldaan.